### PR TITLE
PUD-558: Add email address & phone number to user details

### PR DESCRIPTION
**Pull request by philsegal**
- use alpine to fix postgres failure to launch on circleci

### CODE REVIEW

**Review by jonnywyatt — DISMISSED** [review]
Does the signature upload need testing?

**Comment by philsegal** [comment]
> Does the signature upload need testing?

we upload a signature already. and the fact the recall notification generates without exception means it 'works'. I think that's enough for e2es

File: docker-compose.yml [hunk 1]
```diff
@@ -102,7 +102,7 @@ services:
       - SESSION_SECRET=session-secret
 
   postgres:
-    image: postgres:13
+    image: postgres:13-alpine
```
Comment (shahnaazr) [inline docker-compose.yml:105]: are we using postgres alpine in api as well?

**Review by shahnaazr — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (philsegal) [reply docker-compose.yml:105]: no, but we can/could change it if necessary

**Pull request merged**